src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1106,7 +1108,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $prefix = ''
 
                                         # Don't show line information if script module
-                                        if (($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.CategoryInfo.Category -eq 'ParserError') -and !($myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))) {
+                                        if (($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.Type -is [System.Management.Automation.ParseException]) -and !($myinv.ScriptName -and $myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))) {
```

Comment:
> !($myinv.ScriptName -and $myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))

Maybe use `!($myinv.ScriptName?.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))` instead?

Reply:
Or, change it to `$myinv.ScriptName -notlike *.psm1`? The `-notlike` use case-insensitive check.

Reply:
This was changed to use `-match`